test-vectors/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+mod peg_handoff;
+mod peg_in;
+mod peg_out;
+mod utils;
```

Comment:
fix: These modules should be public to expose the functions
```suggestion
pub mod peg_handoff;
pub mod peg_in;
pub mod peg_out;
pub mod utils;
```